JJG/Ping.php

```diff
@@ -194,6 +194,8 @@ public function ping($method = 'exec') {
       case 'socket':
         $latency = $this->pingSocket();
         break;
+      default:
+        throw new \InvalidArgumentException('unsupported ping method');
```

Comment:
Since this function could throw an exception, can you add a `@throws` doc line in the function documentation (see: https://phpdoc.org/docs/latest/references/phpdoc/tags/throws.html).